calibration_sim/pico_scanning.par

```diff
@@ -0,0 +1,30 @@
+--samplerate
+1000.0
+--spinperiod
+60.0
+--spinangle
+73.0
+--precperiod
+14400.0
+--precangle
+22.0
+--hwprpm
+0.0
+--baseline
+60.0
+--nside
+1024
+--obs
+24.0
+--input_pysm_model
+a1,d1,f1,s1
```

Comment:
@ziotom78, are you fine with the suggestion of @jdborrill to have: 

> unlensed scalar CMB (r=0) with an intrinsic dipole

? can you take care of producing it?

Reply:
Sure, I'll do it immediately.

Reply:
Done, CAMB output is in `/scratch1/scratchdirs/tomasi/PICO_simulations/cmb_sky`. What is of interest for this simulation is the CMB sky map (with NSIDE=1024), saved in `camb_20960065_map.fits`.

Reply:
thanks @ziotom78  can you please make a pull request on my `pico_calibration_sim` branch and add a subfolder `cmb_sky` with the CAMB configuration (& scripts if useful)?

Reply:
@zonca, I used the web interface at https://lambda.gsfc.nasa.gov/toolbox/tb_camb_form.cfm, and I did not prepare any configuration file. Which files should I add to the repository? The text file with all the alms? The NSIDE=1024 map is perhaps too large to be uploaded.

Reply:
ok, the alms are fine. too bad that the form doesn't allow to download a text configuration file that can be used to run the CAMB executable from the terminal...